OWNERS_ALIASES

```diff
@@ -142,6 +142,12 @@ aliases:
     - jeremyrickard
     - liggitt
     - micahhausler
+  wg-node-lifecycle-leads:
+    - atiratree
+    - fabriziopandini
+    - humblec
+    - kwilczynski
+    - rthallisey
   wg-policy-leads:
```

Comment:
I'm interested in helping here as well

Reply:
Great, thanks for the help @haircommander! Added.